apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -349,7 +350,14 @@ const TutorialExplorer = React.createClass({
 
         return filterGroupsSatisfied;
       }).sort((tutorial1, tutorial2) => {
-        if (sortBy === TutorialsSortBy.popularityrank) {
+        let useSortBy;
+        if (sortBy === TutorialsSortBy.default) {
+          useSortBy = sortByPopularity ? TutorialsSortBy.popularityrank : TutorialsSortBy.displayweight;
+        } else {
+          useSortBy = sortBy;
+        }
```

Comment:
I'm having a little bit of trouble following this. (I think part of this might be because `sortBy` reads as a noun instead of a verb - I found it easier to parse if I mentally replaced it with `sortMethod`). What I think this is saying:
- If we haven't been provided a sorting method, look at sortByPopularity to determine our sorting method.
- If we have been provided a sorting method, use that

Does this mean we won't actually ever end up with the default sorting method? Would it be better called `TutorialsSortBy.unset`?